### PR TITLE
Fixed exercises

### PR DESCRIPTION
**Pull request by Immortalgleam**

### CODE REVIEW

File: Exercises.py [hunk 1]
```diff
@@ -1,18 +1,17 @@
-#                    Exercises
 # 1. Index of the Maximum Element:
 # Create a list of numbers and find the index of the maximum element in the list.
```
Comment (kryvetskyi) [inline Exercises.py]: remove this empty line in all tasks

File: Exercises.py [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 print(max_index)
 
 # 2. String Concatenation:
 # Ask the user for their first name and last name, then concatenate them into a single string.
 
-first_name = 'Tolik'
-last_name = 'Kryveckiy'
+first_name = 'John'
+last_name = 'Dou'
 print(first_name, last_name)
```
Comment (kryvetskyi) [inline Exercises.py:11]: remove

File: Exercises.py [hunk 3]
```diff
@@ -23,23 +22,23 @@
 # 4. Checking Element Presence:
 # Create a list and ask the user to input a number, then check if this number is present in the list.
 
-l = [1, 2, 3]
-print(l[1])
-if l[1] == 2:
-    print('number is present')
+list_numbers = [1, 2, 3]
+user_number = int(input("Please enter the number: ")) # input() function
+if user_number in list_numbers:
```
Comment (kryvetskyi) [inline Exercises.py:23]: only check if number is in a list
result should be of type boolean

File: Exercises.py [hunk 4]
```diff
@@ -23,23 +22,23 @@
...
 
-print(first_name is last_name)
+print(first_name == last_name) # equality of 2 objects
 
 # 6. Sum of Tuple Elements:
 # Create a tuple of numbers and print their sum.
 
-new_tuple = 1, 3, 4, 5  # or we can use round brackets ()
+new_tuple = (1, 3, 4, 5)
 print(new_tuple)
```
Comment (kryvetskyi) [inline Exercises.py:35]: remove

File: Exercises.py [hunk 5]
```diff
@@ -92,11 +91,9 @@
 
 
 # 10. Create a string of length 10 with empty spaces in it. change empty spaces with '+' sign
-
 str_new = ' ' * 10
-str_new = '+' * 10
+str_new = str_new.replace('', '+') # string method to replace empty spaces with '+'
```
Comment (kryvetskyi) [inline Exercises.py]: check the result
yo did not replaced empty spaces

**Review by kryvetskyi — CHANGES_REQUESTED** [review]

File: Exercises.py (hunk 3 above)
Comment (Immortalgleam) [reply Exercises.py:23]: fixed

File: Exercises.py (hunk 5 above)
Comment (Immortalgleam) [reply Exercises.py]: fixed

File: Exercises.py (hunk 1 above)
Comment (Immortalgleam) [reply Exercises.py]: done

File: Exercises.py (hunk 2 above)
Comment (Immortalgleam) [reply Exercises.py:11]: done

File: Exercises.py (hunk 4 above)
Comment (Immortalgleam) [reply Exercises.py:35]: done

**Review by kryvetskyi — APPROVED** [review]

**Comment by Immortalgleam** [comment]
closed

**Pull request closed without merging**